src/main.ts

```diff
@@ -49,7 +49,7 @@ import type {
 } from './types';
 import { generateTable } from './table';
 import { createShareableUrl } from './share';
-import {filterTests} from './commands/eval/filterTests';
+import { filterTests } from './commands/eval/filterTests';
```

Comment:
Sorry about this.

Could add `bracketSpacing: true` to `.prettierrc.yaml` I use Prettier format on save. Surprised it didn't pick this up.

Reply:
:+1:  Done in ba8da4ec